tests/xtro-sharpie/api-annotations-dotnet/common-WebKit.ignore

```diff
@@ -2,7 +2,3 @@
 !missing-selector! WKWebView::findInteraction not bound
 !missing-selector! WKWebView::isFindInteractionEnabled not bound
 !missing-selector! WKWebView::setFindInteractionEnabled: not bound
-
-## https://github.com/xamarin/xamarin-macios/issues/15593
-!missing-selector! WKWebpagePreferences::isLockdownModeEnabled not bound
-!missing-selector! WKWebpagePreferences::setLockdownModeEnabled: not bound
```

Comment:
Nitpick: missing end of line.

Reply:
I think the missing EOL symbol is from the before changes. The new changes includes an EOL.

Reply:
-EMONDAYMORNING 🤦